doc/sphinx/installation.rst

```diff
@@ -87,22 +87,42 @@ Optionally the ccmake utility can be installed for easier configuration:
 Installing Requirements on Mac OS X
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To make ESPResSo run on Mac OS X 10.9 or higher, its dependencies can be
+To make |es| run on Mac OS X 10.9 or higher, its dependencies can be
 installed using MacPorts. First, download the installer package
 appropriate for your Mac OS X version from
 https://www.macports.org/install.php and install it. Then, run the
 following commands:
 
 .. code-block:: bash
 
-    sudo xcode-select –install sudo xcodebuild -license accept
+    sudo xcode-select --install
+    sudo xcodebuild -license accept
     sudo port selfupdate
     sudo port port install cmake python27 python27-cython python27-numpy \
```

Comment:
Since you are touching the file, remove the doubled ``port`` command

Reply:
Thanks, done.